.github/workflows/cli-ci.yml

```diff
@@ -0,0 +1,89 @@
+name: cli/ubi CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'cli/go.mod'
+      - 'cli/ubi.go'
+      - 'bin/ubi'
+      - 'spec/cli_spec.rb'
+  pull_request:
+    paths:
+      - 'cli/go.mod'
+      - 'cli/ubi.go'
+      - 'bin/ubi'
+      - 'spec/cli_spec.rb'
+
+jobs:
+  cli-ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubicloud, ubicloud-arm]
+    name: cli/ubi CI - ${{matrix.runs-on}}
+    runs-on: ${{matrix.runs-on}}
+
+    env:
+      DB_USER: clover
+      DB_PASSWORD: nonempty
+      DB_NAME: clover_test
+
+    services:
+      postgres:
+        image: postgres:15.4
+        env:
+          POSTGRES_USER: ${{ env.DB_USER }}
+          POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
+          POSTGRES_DB: ${{ env.DB_NAME }}
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - name: Perform superuser-only actions, then remove superuser
+      run: |
+        psql "postgres://${{ env.DB_USER }}:${{ env.DB_PASSWORD }}@localhost:5432/${{ env.DB_NAME }}" \
+          -c "CREATE EXTENSION citext; CREATE EXTENSION btree_gist; CREATE ROLE clover_password PASSWORD '${{ env.DB_PASSWORD }}' LOGIN; ALTER ROLE ${{ env.DB_USER }} NOSUPERUSER"
+
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Install ruby for ARM runners if not cached
```

Comment:
Actually, `ruby/setup-ruby` has added prebuilt artifacts for arm64 as well recently.
https://github.com/ruby/setup-ruby/issues/689#issuecomment-2600265901

We probably don't need it anymore, so we might remove it from our Clover CI too.

Reply:
I removed it from cli-ci.yml, and CI still passed.  I'll submit a PR for ci.yml in a bit.